### DISCUSSION TITLE
2.21.0版本之后部分多语言插件不生效了

### DISCUSSION THREAD

**Discussion by shilin8805** [opensumi/core#2317] [Q&A] [answered]
升级到2.21.0版本之后vscode-language-pack-zh-hans这种多语言汉化的插件不生效了，可以看下git那部分，还是英文的。之前的版本是没问题的

**Comment by Ricbet** [comment]
看下你的 git 版本和 vscode-language-pack-zh-hans 版本是不是一致的

**Reply by shilin8805** [reply]
一致的~我看2.21.0版本有个逻辑优化，不知道是不是这块影响的
<img width="1669" alt="image" src="https://user-images.githubusercontent.com/3340210/220091290-e2592ff7-d629-43a0-aff7-2b56055f64b6.png">

**Reply by erha19** [reply]
@shilin8805 麻烦提供一下你使用的插件及版本号，我们在 2.21 版本尝试复现一下，另外，可以看一下是否使用了2.21 最新的版本，可能是有问题的中间版本（或尝试一下最新 2.22.4）。

**Reply by shilin8805** [reply]
装的插件版本：vscode-language-pack-zh-hans（1.69.0）、git（1.62.3）、git-ui（1.55.2）、git-base（1.64.0-next.68976705968）
我试了2.22.4也不行，一直试到2.20.13版本才可以，2.21.0之后的版本都不行。

**Reply by Ricbet** [reply]
试试 vscode-language-pack-zh-hans（1.75.0）版本呢？

**Reply by shilin8805** [reply]
试过了，也不行

**Comment by erha19** [comment]
@Ricbet 这个问题跟进看一下

**Comment by erha19** [comment]
@shilin8805 已修复 https://github.com/opensumi/core/pull/2382 ，可以看一下分支内发布的测试版本，或等待 2.22.7 修复版本发布

**Reply by shilin8805** [reply]
谢谢

**Reply by shilin8805** [reply]
@erha19 @Ricbet 还是有个小瑕疵。。。SCM面板中的部分区域，多语言插件还是没有生效。试了下，2.21.0之前的版本没问题。。
<img width="357" alt="image" src="https://user-images.githubusercontent.com/3340210/228530812-da0ea96b-c181-4f98-841e-865764e695ec.png">

**Reply by Ricbet** [reply]
可能是这里的输入框 placeholder 没做转换处理，晚点我在 2.23 版本上修复一下

**Reply by Ricbet** [reply]
@shilin8805 已修复, https://github.com/opensumi/core/pulls
你可以先用这个版本试试 `2.23.2-next-1680160947.0`

**Reply by shilin8805** [reply]
这个版本试了。。还是不行。
我看不只是placeholder有问题，那个Tree的标题“Changes”也没有翻译。2.21.0之前的版本我试了都没问题。。

<img width="830" alt="image" src="https://user-images.githubusercontent.com/3340210/228772198-f77166ac-3810-448f-adb5-ab240fcb4b69.png">

**Reply by shilin8805** [reply]
另外我用的vscode翻译插件是1.69.0版本，我看里面有这个对应的key值
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/3340210/228774837-2c2f0a0f-d3a2-438f-98dd-cad9bf89d4f3.png">

**Reply by shilin8805** [reply]
@Ricbet

**Reply by Ricbet** [reply]
@shilin8805 或者你试试翻译插件的 1.62 版本呢，改成和 git 插件一样的版本

**Reply by shilin8805** [reply]
@Ricbet 没有1.62版本的插件，换成1.63版本也不行。。。

**Comment by bytemain** [comment]
https://github.com/opensumi/core/pull/2529

2.23.2-next-1680169656.0

试下这个

**Reply by shilin8805** [reply, marked as answer]
@bytemain @Ricbet 
这个版本 输入框和组件树标题的可以显示中文了，但是菜单上的翻译又没了
<img width="541" alt="image" src="https://user-images.githubusercontent.com/3340210/228824114-db8d373c-aff5-43c5-984a-77959013f1f8.png">
